ccpp/data/GFS_typedefs.F90

```diff
@@ -4787,14 +4874,15 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%kdt              = nint(Model%fhour*con_hr/Model%dtp)
     Model%first_time_step  = .true.
     Model%restart          = restart
+    Model%lsm_cold_start   = .not. restart
```

Comment:
@SamuelTrahanNOAA @ @hu5970 Interesting, that lsm_cold_start is always .false. when the model runs from the restart file. I don't think this is a good default. At least, we should be careful and might need to add lsm_cold_start to input.nml for cycled RRFS.

Reply:
@SamuelTrahanNOAA @hu5970 Some time ago Dom and I discussed a capability to do cold-start atmosphere with the cycled lsm fields. This new logical variable must be related to this capability. We need to make sure that it is used correctly in our version of RRFS.